scalableminds/webknossos
# Miscellaneous improvements to the proofreading tool

- fix race condition in conflict-checking code by re-reading current version from store
- use base mapping name of editable mapping in mapping UI if possible
- don't try to load agglomerate skeleton if user selects a node in proofreading tool (this could load another skeleton if a node was only visible due to a high clipping distance which was confusing)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open DS with agglomerate mapping and select it
- select proofreading tool and use click on a segment
- the left sidebar should show a readable name for the mapping
- select a node in the tree --> nothing else should happen

### Issues:
- contributes to #6288

------
(Please delete unneeded items, merge only when none are left open)
